lib/openshift/route.rb

```diff
@@ -106,6 +106,23 @@ def dns(by: nil, user: nil, cached: true, quiet: false)
       return raw_resource(user: user, cached: cached, quiet: quiet).
         dig("spec", "host")
     end
+
+    def tls_insec_term_policy(by: nil, user: nil, cached: true, quiet: false)
```

Comment:
@yapei logically it looks correct, but I think @akostadinov prefer that we use object-like approace to access the `spec` (@akostadinov please correct me if I have the wrong assumption), which means we would need to make a `RouteSpec` class or something similiar.   I can work on that later today.

Reply:
@yapei, I've added PR #347

Reply:
Thanks very much, I may misunderstood it